### PR TITLE
Fix KA reskin mods and welding tool inhand

### PR DESCRIPTION
**Pull request by Markolie**
Fixes https://github.com/ParadiseSS13/Paradise/issues/6441
I forgot to update the overlay sprite of the kinetic accelerator "empty" sprite.

Fixes https://github.com/ParadiseSS13/Paradise/issues/6440
Fixes https://github.com/ParadiseSS13/Paradise/issues/6444
Welding tools now have a proper in-hand when turned on. 

🆑 Markolie
fix: The kinetic accelerator sprite modkits now apply properly when the kinetic accelerator is empty.
fix: Welding tools now once again have a proper in-hand when turned on.
/🆑

### CODE REVIEW

**Comment by uraniummeltdown** [comment]
Please re-add the welder item_state s from https://github.com/ParadiseSS13/Paradise/pull/6395/commits/1edc9a63e8fa5c51a0dfd6be87328e9b2138d707 that I stupidly removed, also please remove the ```icon_state = "welder"``` line i wrote under Industrial Welding Tool

**Comment by Markolie** [comment]
Done.

File: code/game/objects/items/weapons/tools.dm [hunk 1]
```diff
@@ -151,10 +152,9 @@
 	create_reagents(max_fuel)
 	reagents.add_reagent("fuel", max_fuel)
 	update_icon()
-	return
 
 /obj/item/weapon/weldingtool/examine(mob/user)
-	if(..(user, 0))
+	if(..(user, 1))
```
Comment (Crazylemon64) [inline code/game/objects/items/weapons/tools.dm]: I'm pretty sure this increases the radius of examination from same-tile to adjacent-tile, but that's no biggy. Was it allowing examination from elsewhere before?

Comment (Markolie) [inline code/game/objects/items/weapons/tools.dm]: Does it? I thought that setting the second parameter to 1 means that this examination text only shows up if the user is on the same tile. If it's set to 0 I thought it always showed up.

Let me double check this.

**Pull request merged**